app/scripts/generate_user_email_list.py

```diff
@@ -0,0 +1,96 @@
+import csv
+import sys
+import multiprocessing
+from multiprocessing.pool import ThreadPool
+
+from dmutils.apiclient import DataAPIClient
+from dmutils.apiclient.errors import HTTPError
+from dmutils.audit import AuditTypes
+
+
+def selection_status(client, framework_id):
+    def something(user):
```

Comment:
There is a method somewhere in supplier frontend that encapsulates this same logic (see https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/main/helpers/frameworks.py#L186).  Might be more hassle than it's worth trying to share it for use here though I guess.